test/helpers/kubectl.go

```diff
@@ -1404,6 +1412,35 @@ func (kub *Kubectl) ciliumInstallHelm(filename string, options map[string]string
 	return nil
 }
 
+func (kub *Kubectl) waitToDeleteCilium() error {
```

Comment:
I haven't looked but it may be possible to approximate this with one of the WaitNPods type functions. Those may return when >=N is true, though, so that wouldn't work.

Reply:
I moved this function from `test/k8sT/assertionHelpers.go`.